### ISSUE TITLE
ERROR: 1opHWsHr798: "token" parameter not in video info for unknown reason;

### ISSUE THREAD

**Issue by flajann2** [ytdl-org/youtube-dl#21765]
## Checklist

- [ x] I'm reporting a broken site support issue
- [ x] I've verified that I'm running youtube-dl version **2019.07.12**
- [ x] I've checked that all provided URLs are alive and playable in a browser
- [x ] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [ x] I've searched the bugtracker for similar bug reports including closed ones
- [x ] I've read bugs section in FAQ

## Verbose log

```
λ youtube-dl -v https://youtu.be/1opHWsHr798
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-v', u'https://youtu.be/1opHWsHr798']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2019.03.01
[debug] Python version 2.7.16 (CPython) - Linux-5.0.0-20-generic-x86_64-with-Ubuntu-19.04-disco
[debug] exe versions: ffmpeg 4.1.3, ffprobe 4.1.3
[debug] Proxy map: {}
[youtube] 1opHWsHr798: Downloading webpage
[youtube] 1opHWsHr798: Downloading video info webpage
ERROR: 1opHWsHr798: "token" parameter not in video info for unknown reason; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/YoutubeDL.py", line 794, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/extractor/common.py", line 508, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 1689, in _real_extract
    video_id=video_id)
ExtractorError: 1opHWsHr798: "token" parameter not in video info for unknown reason; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

## Description

Basically, when try to download https://youtu.be/1opHWsHr798 , the above error occurs and the download is aborted. I saw similar cases reported in the April or May timeframes, but I think this is "fresh" .

**Comment by Hrxn** [comment]
> [debug] youtube-dl version 2019.03.01

What does that tell us?
